modules/imaging_uploader/php/imaging_uploader.class.inc

```diff
@@ -242,6 +242,7 @@ class Imaging_Uploader extends \NDB_Menu_Filter_Form
      *       .gz, .zip or .tgz
      *  - The fields   CandID, PSCID  and VisitLabel are filled out
      *  - The inputted PSCID,  CandID and VisitLabel exist in the database
+     *  - The inputted candidate is active in the database
```

Comment:
Nitpick: "inputted" sounds weird to me...Maybe change to "The <whatever> that was input...." ?

Reply:
I'm not sure about "The that was input ..".

maybe "The input candidate" or "The candidate inputted"?

Reply:
@zaliqarosli @PapillonMcGill exactly

Reply:
i approve "The candidate that was inputted" lol

Reply:
The past tense of "input" is "input"

Reply:
wow really?? amazing that i'm still learning new things about this language